masterlist.yaml

```
@@ -13737,12 +13742,21 @@ plugins:
       - 'RTT-Weye Relocation Patch.esp'
       - 'VHBloodlines 1.2.esp'
     req: [ *SilentVoice ]
+    msg:
+      - <<: *useVersion
+        subs: [ 'Cobl' ]
+        condition: 'active("Cobl Main.esm") and checksum("AFK_Weye.esp", 5A1114F9)'
```

The version number can be used to differentiate between the non-Cobl & Cobl versions:

```suggestion
        condition: 'active("Cobl Main.esm") and version("AFK_Weye.esp", "2.32.Non-COBL", ==)'
```